dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/csv/DefaultCsvImportService.java

```diff
@@ -437,6 +443,16 @@ public Metadata fromCsv( InputStream input, CsvImportClass importClass )
         return list;
     }
 
+    private void setGeometry( OrganisationUnit ou, FeatureType featureType, String coordinates )
+        throws IOException
+    {
+
+        if ( !featureType.equals( FeatureType.NONE ) && StringUtils.isNotBlank( coordinates ) )
```

Comment:
Use == comparison for enums and to avoid nullpointer vulnerability:

featureType == FeatureType.NONE